docs/keywords.json

```diff
@@ -3738,5 +3738,12 @@
       "remote sensing"
     ],
     "category":"Objects"
-  }
+  },
+	"minus-square": {
+  	"alias": [
```

Comment:
this block should be indented one more level.

Reply:
@paulcpederson is this better?